### DISCUSSION TITLE
[SC]: Mancato collegamento annotazione di divorzio estero su atto di matrimonio estero

### DISCUSSION THREAD

**Discussion by siscomanpr** [italia/ansc#835] [Q&A] [answered]
### Ambiente

produzione

### Servizio

R008 - Servizi cooperativi notifiche

### idOperazione

0

### idEvento o idAnsc

8741

### Provenienza segnalazione

Software House

### Comune di riferimento e operatore

LOANO

### Data del test (yyyy-mm-dd)

18/04/2024

### Descrivi il problema

E' stato trascritto un atto di matrimonio estero ricevuto da consolato relativo a un cittadino AIRE (2024-8707-71-009034) con il caso d'uso 332000.
Subito dopo è stata trascritta la sentenza estera di divorzio estero relativa allo scioglimento del matrimonio sopracitato (2024-8741-72-009034) con il caso d'uso 347000.
Ci aspettavamo un'annotazione automatica generata dal sistema di tipo DIGITALE visto che l'atto di matrimonio era già digitale, invece è arrivata una notifica di tipo "( da inviare via PEC )" con id 2024-8744-00000-009034. 
Potete verificare il motivo del mancato collegamento con l'atto primario di matrimonio? 
Ora il comune deve procedere registrando manualmente l'annotazione sull'atto tramite caso d'uso 366999?

Grazie.

**Comment by mttfranci** [comment, marked as answer]
Buongiorno @siscomanpr riguardo i quesiti posti  :

> Potete verificare il motivo del mancato collegamento con l'atto primario di matrimonio?

La gestione dei soggetti AIRE, in particolari le notifiche e le annotazioni automatiche, fa parte di un intervento in corso, il cui rilascio è previsto a fine maggio. Per l'elenco delle principali attività in via di rilascio a breve potete consultare la  [Roadmap](https://github.com/italia/ansc/blob/main/docs/Roadmap.md).

> Ora il comune deve procedere registrando manualmente l'annotazione sull'atto tramite caso d'uso 366999?

Si, fino al rilascio del suddetto intervento, potete procedere così.

Saluti

Assistenza ANSC
